exp-player/addon/components/exp-overview/component.js

```diff
@@ -107,13 +107,16 @@ export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-overview',
     layout: layout,
     questions: questions,
-    responses: Ember.computed('questions', function() {
-        var questions = this.get('questions');
-        var responses = {};
-        for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
-        }
-        return responses;
+    showOptional: Ember.computed('questions.7.value', function() {
+       return this.questions[7].value === 'global.yesLabel';
+    }),
+    responses: Ember.computed(function() {
```

Comment:
If you're counting on the computed being reevaluated when pressing 'continue' , I'd suggest adding `.volatile()` to the computed definition (http://emberjs.com/api/classes/Ember.ComputedProperty.html#method_volatile)

Or looking at the first example here: https://guides.emberjs.com/v2.6.0/object-model/computed-properties-and-aggregate-data/

you might be able to use `@each.value`